test/test_p4est_2d.jl

```diff
@@ -221,6 +221,34 @@ end
     end
 end
 
+@trixi_testset "elixir_euler_shockcapturing_ec_float32.jl" begin
+    @test_trixi_include(joinpath(EXAMPLES_DIR,
+                                 "elixir_euler_shockcapturing_ec_float32.jl"),
+                        l2=[
+                            0.09539953f0,
+                            0.10563527f0,
+                            0.105637245f0,
+                            0.3507514f0,
+                        ],
+                        linf=[
+                            0.2942562f0,
+                            0.4079147f0,
+                            0.3972956f0,
+                            1.0810697f0,
+                        ],
```

Comment:
Were these values different due to the modifications to the `ln_mean` such that we cannot use `Float32` of the double precision `elixir_euler_shockcapturing_ec.jl` test?

Reply:
I don't know whether it's exactly the `ln_mean` or something else but yes, the `Float64` values from the other test converted to `Float32` do not work locally for me